src/hb-script-lang-table.h

```diff
@@ -0,0 +1,253 @@
+  { "ca",       { HB_SCRIPT_LATIN } },
+  { "ce",       { HB_SCRIPT_CYRILLIC } },
+  { "ch",       { HB_SCRIPT_LATIN } },
+  { "chm",      { HB_SCRIPT_CYRILLIC } },
+  { "chr",      { HB_SCRIPT_CHEROKEE } },
+  { "co",       { HB_SCRIPT_LATIN } },
+  { "crh",      { HB_SCRIPT_LATIN } },
+  { "cs",       { HB_SCRIPT_LATIN } },
+  { "csb",      { HB_SCRIPT_LATIN } },
+  { "cu",       { HB_SCRIPT_CYRILLIC } },
+  { "cv",       { HB_SCRIPT_CYRILLIC, HB_SCRIPT_LATIN } },
+  { "cy",       { HB_SCRIPT_LATIN } },
+  { "da",       { HB_SCRIPT_LATIN } },
+  { "de",       { HB_SCRIPT_LATIN } },
+  { "doi",      { HB_SCRIPT_DEVANAGARI } },
+  { "dv",       { HB_SCRIPT_THAANA } },
+  { "dz",       { HB_SCRIPT_TIBETAN } },
+  { "ee",       { HB_SCRIPT_LATIN } },
+  { "el",       { HB_SCRIPT_GREEK } },
+  { "en",       { HB_SCRIPT_LATIN } },
+  { "eo",       { HB_SCRIPT_LATIN } },
+  { "es",       { HB_SCRIPT_LATIN } },
+  { "et",       { HB_SCRIPT_LATIN } },
+  { "eu",       { HB_SCRIPT_LATIN } },
+  { "fa",       { HB_SCRIPT_ARABIC } },
+  { "fat",      { HB_SCRIPT_LATIN } },
+  { "ff",       { HB_SCRIPT_LATIN } },
+  { "fi",       { HB_SCRIPT_LATIN } },
+  { "fil",      { HB_SCRIPT_LATIN } },
+  { "fj",       { HB_SCRIPT_LATIN } },
+  { "fo",       { HB_SCRIPT_LATIN } },
+  { "fr",       { HB_SCRIPT_LATIN } },
+  { "fur",      { HB_SCRIPT_LATIN } },
+  { "fy",       { HB_SCRIPT_LATIN } },
+  { "ga",       { HB_SCRIPT_LATIN } },
+  { "gd",       { HB_SCRIPT_LATIN } },
+  { "gez",      { HB_SCRIPT_ETHIOPIC } },
+  { "gl",       { HB_SCRIPT_LATIN } },
+  { "gn",       { HB_SCRIPT_LATIN } },
+  { "gu",       { HB_SCRIPT_GUJARATI } },
+  { "gv",       { HB_SCRIPT_LATIN } },
+  { "ha",       { HB_SCRIPT_LATIN } },
+  { "haw",      { HB_SCRIPT_LATIN } },
+  { "he",       { HB_SCRIPT_HEBREW } },
+  { "hi",       { HB_SCRIPT_DEVANAGARI } },
+  { "hne",      { HB_SCRIPT_DEVANAGARI } },
+  { "ho",       { HB_SCRIPT_LATIN } },
+  { "hr",       { HB_SCRIPT_LATIN } },
+  { "hsb",      { HB_SCRIPT_LATIN } },
+  { "ht",       { HB_SCRIPT_LATIN } },
+  { "hu",       { HB_SCRIPT_LATIN } },
+  { "hy",       { HB_SCRIPT_ARMENIAN } },
+  { "hz",       { HB_SCRIPT_LATIN } },
+  { "ia",       { HB_SCRIPT_LATIN } },
+  { "id",       { HB_SCRIPT_LATIN } },
+  { "ie",       { HB_SCRIPT_LATIN } },
+  { "ig",       { HB_SCRIPT_LATIN } },
+  { "ii",       { HB_SCRIPT_YI } },
+  { "ik",       { HB_SCRIPT_CYRILLIC } },
+  { "io",       { HB_SCRIPT_LATIN } },
+  { "is",       { HB_SCRIPT_LATIN } },
+  { "it",       { HB_SCRIPT_LATIN } },
+  { "iu",       { HB_SCRIPT_CANADIAN_SYLLABICS } },
+  { "ja",       { HB_SCRIPT_HAN, HB_SCRIPT_KATAKANA, HB_SCRIPT_HIRAGANA } },
+  { "jv",       { HB_SCRIPT_LATIN } },
+  { "ka",       { HB_SCRIPT_GEORGIAN } },
+  { "kaa",      { HB_SCRIPT_CYRILLIC } },
+  { "kab",      { HB_SCRIPT_LATIN } },
+  { "ki",       { HB_SCRIPT_LATIN } },
+  { "kj",       { HB_SCRIPT_LATIN } },
+  { "kk",       { HB_SCRIPT_CYRILLIC } },
```

Comment:
What is the source of truth for this list and how do we keep it up to date (changes are rare but do happen, for example [Kazakh is witching to Latin, presumably staring this year](https://en.wikipedia.org/wiki/Kazakh_alphabets#Latest_developments).

Reply:
> What is the source of truth for this list and how do we keep it up to date 

Pango ships a C program that uses fontconfigs charset information for each language, and then looks at the script for each Unicode character in the charset. I didn't include that here since, fontconfig is not a dependency of HarfBuzz, and, as you say, this rarely changes.